rocAL/rocAL_pybind/amd/rocal/readers.py

```diff
@@ -34,7 +34,7 @@ def coco(*inputs, file_root, annotations_file='', bytes_per_sample_hint=0, dump_
     labels = []
     bboxes = []
     kwargs_pybind = {"source_path": annotations_file, "is_output":True, "is_box_encoder":is_box_encoder, "is_box_iou_matcher":is_box_iou_matcher }
-    meta_data = b.COCOReader(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    meta_data = b.cocoReader(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
```

Comment:
We seem to have not removed unused args from readers.py file. Please note @SundarRajan28

Reply:
Nothing is changed in readers.py except the reader pybind names. We have another PR where we make the reader function argument changes